.github/workflows/master.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           - '-Pspark-3.0 -Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.1.1 -Dspark.archive.name=spark-3.1.1-bin-hadoop2.7.tgz -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.DataLakeTest'
           - '-Pspark-3.1'
           - '-Pspark-3.1 -Dhadoop.binary.version=3.2'
+    env:
+      SPARK_LOCAL_IP: 127.0.0.1
+      KYUUBI_FRONTEND_BIND_HOST: 127.0.0.1
```

Comment:
How about just add envs and properties in `scalatest-maven-plugin`, add a new env is overkill for me to only fix a CI issue

Reply:
sounds good